File: 240322/학점 계산기/credit-calculator.py
n = int(input())
arr = list(map(float, input().split()))

s = 0
for i in range(n):
    s +=arr[i]
g = s/n
print(f"{g:.1f}")
if g>=4.0:
    print("Perfect")
elif g>=3.0:
    print("Good")
else:
    print("Poor")

File: 240626/학점 계산기/credit-calculator.js
const fs = require("fs")
let input = fs.readFileSync(0).toString().trim().split("\n")

let n = Number(input[0])
let arr = input[1].split(" ").map(Number)

let sum = 0
for(let i =0; i<n; i++){
    sum+=arr[i]
}
let avg = (sum/n).toFixed(1)

console.log(avg)

if(avg>=4.0){
    console.log("Perfect")
}else if(avg>=3.0){
    console.log("Good")
}else{
    console.log("Poor")
}